hummingbot/market/kraken/kraken_constants.py

```diff
@@ -24,10 +24,14 @@
     "AUD"
 ]
 
-FIAT_QUOTES = FIAT_QUOTES + ["Z" + quote for quote in FIAT_QUOTES]
+FIAT_QUOTES = ["Z" + quote for quote in FIAT_QUOTES] + FIAT_QUOTES
 
 QUOTES = CRYPTO_QUOTES + ADDED_CRYPTO_QUOTES + FIAT_QUOTES
 
+SPECIAL_BASES = ["XBT", "ETH", "XRP"]
+
+SPECIAL_QUOTES = ["USD", "USD.d", "EUR", "EUR.d", "JYP", "JYP.d", "CAD", "CAD.d", "GBP", "GBP.d"]
```

Comment:
Hi @vic-en the correct fiat token is JPY instead of JYP.

```
2020-06-28 08:04:06,001 - 9594 - hummingbot.core.clock - ERROR - Unexpected error running clock tick.
Traceback (most recent call last):
  File "hummingbot/core/clock.pyx", line 119, in hummingbot.core.clock.Clock.run_til
    child_iterator.c_tick(self._current_tick)
  File "hummingbot/strategy/pure_market_making/pure_market_making.pyx", line 547, in hummingbot.strategy.pure_market_making.pure_market_making.PureMarketMakingStrategy.c_tick
    proposal =self.c_create_base_proposal()
  File "hummingbot/strategy/pure_market_making/pure_market_making.pyx", line 572, in hummingbot.strategy.pure_market_making.pure_market_making.PureMarketMakingStrategy.c_create_base_proposal
    price = self.c_get_mid_price() * (Decimal("1") - self._bid_spread - (level * self._order_level_spread))
  File "hummingbot/strategy/pure_market_making/pure_market_making.pyx", line 299, in hummingbot.strategy.pure_market_making.pure_market_making.PureMarketMakingStrategy.c_get_mid_price
    mid_price = self._market_info.get_mid_price()
  File "/home/ubuntu/hummingbot/hummingbot/strategy/market_trading_pair_tuple.py", line 33, in get_mid_price
    return self.market.get_mid_price(self.trading_pair)
  File "hummingbot/market/market_base.pyx", line 109, in hummingbot.market.market_base.MarketBase.get_mid_price
    return (self.get_price(trading_pair, True) + self.get_price(trading_pair, False)) / Decimal("2")
  File "hummingbot/market/market_base.pyx", line 316, in hummingbot.market.market_base.MarketBase.get_price
    return self.c_get_price(trading_pair, is_buy)
  File "hummingbot/market/market_base.pyx", line 215, in hummingbot.market.market_base.MarketBase.c_get_price
    return self.c_quantize_order_price(trading_pair, top_price)
  File "hummingbot/market/market_base.pyx", line 190, in hummingbot.market.market_base.MarketBase.c_quantize_order_price
    price_quantum = self.c_get_order_price_quantum(trading_pair, price)
  File "hummingbot/market/kraken/kraken_market.pyx", line 1124, in hummingbot.market.kraken.kraken_market.KrakenMarket.c_get_order_price_quantum
    TradingRule trading_rule = self._trading_rules[trading_pair]
KeyError: 'XBTJPY'
```

Reply:
Done!